sdk/iOS/RunInstruments.sh

```diff
@@ -41,7 +46,7 @@ fi
 
 unzip -o sdk.zip
 
-xcodebuild -sdk iphonesimulator8.3 || exit 1
+xcodebuild -sdk iphonesimulator9.1 || exit 1
```

Comment:
long term we need to bake this into a param, be nice to be running more than 1 version of sims